zerver/migrations/0274_migrate_alert_word.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+from django.db import migrations
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+import ujson
+
+def move_to_seperate_table(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    UserProfile = apps.get_model('zerver', 'UserProfile')
+    AlertWord = apps.get_model('zerver', 'AlertWord')
+
+    for user_profile in UserProfile.objects.all():
+
+        list_of_words = ujson.loads(user_profile.alert_words)
+        AlertWord.objects.bulk_create(
+            AlertWord(user_profile=user_profile, word=word, realm=user_profile.realm) for word in list_of_words)
+
+def move_back_to_user_profile(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    AlertWord = apps.get_model('zerver', 'AlertWord')
+
+    user_profile_and_words = AlertWord.objects.all().values("user_profile", "word")
+
+    for profile_and_word in user_profile_and_words:
+        user_profile = profile_and_word["user_profile"]
+
+        words = ujson.loads(user_profile.alert_words)
+        words.extend(profile_and_word["word"])
+
+        user_profile.alert_words = ujson.dumps(words)
+        user_profile.save(update_fields=['alert_words'])
```

Comment:
I think if you group these by user_profile first, you could just do a single `save` on each UserProfile object, rather than one per alert word.